docs/components/camera/esp32-camera.md

```diff
@@ -0,0 +1,192 @@
+    "pin_d7": 19,
+    "pin_sccb_sda": 25,
+    "pin_href": 26,
+    "pin_sccb_scl": 23,
+    "sccb_i2c_port": -1,
+    "pin_d1": 35,
+    "pin_d0": 32,
+    "pin_xclk": 27,
+    "pin_reset": 15,
+    "pin_pclk": 21,
+    "pin_d2": 34,
+    "xclk_freq_hz": 20000000,
+    "ledc_channel": 1
+  }
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% alert title="Note" color="note" %}}
+While the following attributes marked as **Optional** do have defaults, it is still recommended that you configure them to your datasheet and device's needs (which likely do not align with the defaults) in order to get the camera stream.
+{{% /alert %}}
+
+The following attributes are available for `esp32-camera` cameras:
+
+<!-- prettier-ignore -->
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| `pin_pwdn` | int | Optional | GPIO pin for camera power down line. <br> Default: -1 |
+| `pin_reset` | int | Optional | GPIO pin for camera reset line. <br> Default: -1 |
+| `pin_xclk` | int | Optional | GPIO pin for camera XCLK line. <br> Default: 21 |
+| `pin_sccb_sda` | int | Optional | GPIO pin for camera SDA line. <br> Default: 26 |
+| `pin_sccb_scl` | int | Optional | GPIO pin for camera SCL line. <br> Default: 27 |
+| `pin_d7` | int | Optional | GPIO pin for camera D7 line. <br> Default: 35|
+| `pin_d6` | int | Optional | GPIO pin for camera D6 line. <br> Default: 34 |
+| `pin_d5` | int | Optional | GPIO pin for camera D5 line. <br> Default: 39 |
+| `pin_d4` | int | Optional | GPIO pin for camera D4 line. <br> Default: 36 |
+| `pin_d3` | int | Optional | GPIO pin for camera D3 line. <br> Default: 19 |
+| `pin_d2` | int | Optional | GPIO pin for camera D2 line. <br> Default: 18 |
+| `pin_d1` | int | Optional | GPIO pin for camera D1 line. <br> Default: 5 |
+| `pin_d0` | int | Optional | GPIO pin for camera D0 line. <br> Default: 4 |
+| `pin_vsync` | int | Optional | GPIO pin for camera VSYNC line. <br> Default: 25 |
+| `pin_href` | int | Optional | GPIO pin for camera HREF line. <br> Default: 23 |
+| `pin_pclk` | int | Optional | GPIO pin for camera PLCK line. <br> Default: 22 |
+| `xclk_freq_hz` | int | Optional | Frequency of XCLK signal, in Hz. <br> Experimental: Set to 16MHz on ESP32-S2 or ESP32-S3 to enable EDMA mode. <br> Default: 20000000 |
+| `ledc_timer` | int | Optional | LEDC timer to generate XCLK. <br> Default: 1 |
+| `ledc_channel` | int | Optional | LEDC channel to generate XCLK. <br> Default: 1 |
+| `frame_size` | int | Optional | Size of the output image. <br> Default: 1 |
+| `jpeg_quality` | int | Optional | Quality of JPEG output. Lower means higher quality. <br> Range: 0-63 <br> Default: 32 |
+
+## View the camera stream
+
+{{< readfile "/static/include/components/camera-view-camera-stream.md" >}}
+
+## Next steps
+
+Configure [data capture](/services/data/capture/) and [cloud sync](/services/data/cloud-sync/) for `viam-micro-server` to capture images from the camera stream and store them in the Viam cloud.
```

Comment:
Offer it as an option - because it's not necessarily the next step. And better to link to the how to

```suggestion
If you need to capture image data and sync it to the cloud, see:

{{< cards >}}
{{% card link="/how-tos/image-data/" %}}
{{< /cards >}}
```